### PR TITLE
[CIS-1077] Add ChannelVisibilityEventMiddleware pendingChanges test

### PR DESCRIPTION
**Pull request by b-onc**
This middleware should be able to see un-saved data from EventDataProcessorMiddleware so it can edit that data.

Check Jira ticket for more info.

#skip_changelog

### CODE REVIEW

**Review by tbarbugli — COMMENTED** [review]
I dont really get this PR, it adds one test and CI is failing

**Comment by b-onc** [comment]
@tbarbugli the test this PR adds doesn't fail, it's another flaky test. the discussed is in the jira ticket / slack

File: Sources/StreamChat/WebSocketClient/EventMiddlewares/ChannelVisibilityEventMiddleware_Tests.swift [hunk 1]
```diff
@@ -66,6 +66,28 @@ final class ChannelVisibilityEventMiddleware_Tests: XCTestCase {
         // Assert `ChannelTruncatedEvent` is forwarded even though database error happened.
         XCTAssertTrue(forwardedEvent is ChannelVisibleEvent)
     }
+    
+    func test_middlewareCanSeePendingEntities() throws {
```
Comment (DominikBucher12) [inline Sources/StreamChat/WebSocketClient/EventMiddlewares/ChannelVisibilityEventMiddleware_Tests.swift:70]: nit: doesn't follow convention :trollface: ->  `test_middleware_canSeePendingEntities`

**Review by DominikBucher12 — APPROVED** [review]
:shipit:

**Pull request merged**